libr/util/ubase64.c

```diff
@@ -47,7 +47,7 @@ static int b64_decode(const char in[4], ut8 out[3], int isz) {
 
 R_API int r_base64_decode(ut8 *bout, const char *bin, int len) {
 	int in, out, ret;
-	if (len < 0) {
+	if (len <= 0) {
```

Comment:
this looks wrong to me. if you pass len = 0 then the buffer should be of length 0, so nothing to encode.

if you pass -1, it should do the strlen. so imho this pr is wrong

what is the issue that you have?